packages/web-react/src/components/Icon/Icon.tsx

```diff
@@ -19,6 +22,14 @@ export const Icon = (props: IconProps) => {
     icon = `<title>${title}</title>${icon}`;
   }
 
+  // @deprecated Usage of `html-react-parser` will be required in the next major version.
```

Comment:
This shouldn't be here. I have already removed this.

Reply:
I probably failed some rebase, should be fixed.